packages/wrangler/src/__tests__/helpers/mock-workers-subdomain.ts

```diff
@@ -79,18 +79,21 @@ export function mockUpdateWorkerSubdomain({
 	env,
 	legacyEnv = false,
 	expectedScriptName = "test-name",
+	flakes = 0,
 }: {
 	enabled: boolean;
 	previews_enabled?: boolean;
 	env?: string | undefined;
 	legacyEnv?: boolean | undefined;
 	expectedScriptName?: string;
+	flakes?: number;
```

Comment:
maybe add a comment that this is returning error responses
nit: rename to numFlakes